bugzilla2github

```diff
@@ -617,8 +568,7 @@ def github_issues_add(issues):
             # Check if the issue was imported from the Bugzilla
             github_issue = github_issue.json()
             if ("bugzilla bug id: %d\n" % id).lower() in github_issue["body"].lower():
-                print "Issue #%d already imported, updating..." % id
-                github_issue_update(issue)
+                print "Issue #%d already imported..." % id
```

Comment:
Here the check that the issue whether the issue was already imported is still there. If it is the case, we don't have to update it because it was imported fully at once.